_includes/manuals/2.1/2.1_quickstart_installation.md

```diff
@@ -1,6 +1,6 @@
 [The Foreman installer](/manuals/{{page.version}}/index.html#3.2ForemanInstaller) uses Puppet **(4 or later required)** to install Foreman. This guide assumes that you have a newly installed operating system, on which the installer will setup Foreman, a Puppet master, and the [Smart Proxy](/manuals/{{page.version}}/index.html#4.3SmartProxies) by default. It's **not advisable** to follow the steps below on an existing system, since the installer will affect the configuration of several components.
 
-**If you're looking for a special scenario such as Katello content management, please follow the [Katello]({{ site.baseurl }}plugins/katello) installation instructions**
+**If you want to manage content with Foreman or you're a new user unfamiliar with all of Foreman's capabilities, please follow the [Katello]({{ site.baseurl }}plugins/katello) installation instructions. If you do not install Katello with Foreman, you cannot add it at a later stage**
```

Comment:
Perhaps we should wrap it in a colorful box like we use in other places to make sure it isn't missed: 
```suggestion
<div class="alert alert-info">
If you want to manage content with Foreman or you're a new user unfamiliar with all of Foreman's capabilities, please follow the [Katello]({{ site.baseurl }}plugins/katello) installation instructions instead. If you do not install Katello with Foreman, you cannot add it at a later stage.
</div>
```